swagger.yaml

```diff
@@ -2504,18 +2535,27 @@ components:
           example: EUR
     AddOn:
       type: object
+      required:
+        - add_on
       properties:
         add_on:
           $ref: '#/components/schemas/AddOnObject'
     AddOns:
       type: object
+      required:
+        - add_ons
       properties:
         add_ons:
           type: array
           items:
             $ref: '#/components/schemas/AddOnObject'
     AddOnInput:
       type: object
+      required:
+        - name
+        - code
+        - amount_cents
+        - amount_currency
```

Comment:
@lorddaedra Thanks a lot for the PR. Since we share the same input structure (AddOnInput) for both POST and PUT, we should not require attributes here. We want to allow consumer to update only certain attribute. The same comment applies for all collections.

Reply:
There is an error on this one, @lovrocolic is on it!

Reply:
@lorddaedra https://doc.getlago.com/docs/api/coupons/update-coupon is fixed now